### ISSUE TITLE
Execute PyInstaller release package 0.24.3: 2020-02-01 fails on fresh Gallium OS

### ISSUE THREAD

**Issue by white-gecko** [AKSW/QuitStore#267]
Why I try to execute the quit store on my fresh Gallium OS 3.1 using the executable produced with PyInstaller (version 
0.24.3: 2020-02-01) I get the following error:

```
Traceback (most recent call last):
  File "quit/run.py", line 8, in <module>
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "/root/.pyenv/versions/3.7.5/lib/python3.7/site-packages/PyInstaller/loader/pyimod03_importers.py", line 623, in exec_module
  File "quit/application.py", line 4, in <module>
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "/root/.pyenv/versions/3.7.5/lib/python3.7/site-packages/PyInstaller/loader/pyimod03_importers.py", line 623, in exec_module
  File "quit/conf.py", line 4, in <module>
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "/root/.pyenv/versions/3.7.5/lib/python3.7/site-packages/PyInstaller/loader/pyimod03_importers.py", line 623, in exec_module
  File "site-packages/pygit2/__init__.py", line 268, in <module>
  File "site-packages/pygit2/settings.py", line 63, in __init__
  File "site-packages/pygit2/settings.py", line 71, in _initialize_tls_certificate_locations
  File "site-packages/pygit2/settings.py", line 164, in set_ssl_cert_locations
_pygit2.GitError: OpenSSL error: failed to load certificates: error:00000000:lib(0):func(0):reason(0)
[6796] Failed to execute script run
```

If somebody else gets this error, please comment and describe the system. This will help finding a solution.

**Comment by splattater** [comment]
I can confirm the same behaviour using Ubuntu 20.04 (0.24.5 and 0.24.2 are working).

**Comment by white-gecko** [comment]
Fixed with new build in 0.24.5
